### PR TITLE
[#160505405] Add different designs for the NotFound message

### PR DESCRIPTION
**Pull request by kimotho-njoki**
## What does this PR do?
Shows a different design for empty states

## Description of Task to be completed?
- Refactor the `ItemsNotFoundComponent` to include a different design
- Refactor various components such as `Users`, `Allocations` e.t.c. to display the new design
- Refactor tests affected by changes

## How should this be manually tested?
Head onto `IncidenceReports` page which currently has no content to view the new design

## What are the relevant pivotal tracker stories?
[#160505405](https://www.pivotaltracker.com/n/projects/2146417/stories/160505405)

## Any background context you want to add?
N/A

## Important notes
N/A

## Packages installed
N/A

## Deployment note
N/A

## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()
## Todos
- [x] Raise PR
- [ ] Test
- [ ] Documentation
## Screenshots
screen shot 2018-10-17 at 15 40 39

- Changes made to font
screen shot 2018-10-19 at 11 27 40

### CODE REVIEW

File: src/_css/ItemsNotFoundComponent.scss [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css?family=Mali');
```
Comment (sachungo) [inline src/_css/ItemsNotFoundComponent.scss]: My advice is that use the font-family we currently have in the repository for uniformity across the app.  Ask Arthur about the font family being used

**Comment by kimotho-njoki** [comment]
@sachungo made changes to the font, removed the url that was making hound complain, but then I do not understand why the tests are failing on circle ci but passing locally, please advice.

**Comment by sachungo** [comment]
@kimotho-njoki Please merge this before it becomes stale

**Comment by sachungo** [comment]
Merging as GitHub has disabled the merge the button for Grace

**Pull request merged**